src/shapes/Placemark.js

```diff
@@ -502,6 +504,17 @@ define([
             return this;
         };
 
+        /**
+         * Gets the active attributes for the current distance to the camera and highlighted state.
```

Comment:
```suggestion
         * Set the placemark attributes for the current distance to the camera and highlighted state.
```

Reply:
@Sufaev I can't commit the comment suggestion in your repo.  If you can make the commit, then I'll merge the PR.